entity-client-api/src/main/java/org/terracotta/entity/EntityClientEndpoint.java

```diff
@@ -44,7 +44,7 @@
    * 
    * @param delegate The delegate to use for events originating within the receiver.
    */
-  void setDelegate(EndpointDelegate delegate);
```

Comment:
This set method looks like an implementation detail and probably should be removed form the API